### ISSUE TITLE
Zoom events ignored

### ISSUE THREAD

**Issue by ondrejj** [flot/flot#1719]
The last working zoom is in version 3.2.10. In versions 3.2.11 to 3.2.13 zoom request is ignored, no selection is present and zoom is not done. Downgrade to 3.2.10 works for me as a workaround.

**Comment by ondrejj** [comment]
Still not working with 4.1.1. Any progress here? Do you need something from me?

**Comment by gorkovv** [comment]
Similarly, I'm also waiting for it to be fixed.

**Comment by ondrejj** [comment]
Looks like this is fixed in 4.2.1. Many thanks.

People, can anybody check and then close this?

**Comment by stepand76** [comment]
It seems that since 3.2.11 the flot.selection.js plugin depends on the jquery.event.drag.js.

**Comment by cfzumbrun** [comment]
Still requires jquery.event.drag.js in 4.2.1 for me

**Comment by lelsteph** [comment]
Same Problem here, cannot get the plotselected event except on rare occasions. Any clue? I am using the latest version I just downloaded.

**Comment by ondrejj** [comment]
@lelsteph, which one is latest for you? Github main page shows v0.8.3 as latest, but releases page has v4.2.1.
Try to specify version and download from https://github.com/flot/flot/releases .
I have no similar issues with v4.2.1.

**Comment by lelsteph** [comment]
@ondrejj: Yes 4.2.1. I actually upgraded from v0.8.3. I had no issue with v0.8.3. BTW when you read the version from plot v4.2.1 is actually v3.0.0...

**Comment by lelsteph** [comment]
Actually the behavior is very strange, randomly I can get the event and then zoom once. Then this is working once zoomed!

**Comment by lelsteph** [comment]
Hello
I checked the versions here https://github.com/flot/flot/releases and tested. I found all is working until 3.2.10 included. Starting 3.2.11 I have the problem!
3.2.11 Revision
Fixing issue with selection plugin being used with in conjunction with other plugins using drag events, where we couldn't properly disable the selection plugin drag handlers from the drag handlers of a separate plugin..
Steph
